### ISSUE TITLE
[Bug]: [major compaction]  The "complete" state of the interface "get_compaction_state" does not include the index time for the new segments after major compaction

### ISSUE THREAD

**Issue by binbinlv** [milvus-io/milvus#32311]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: lru_dev branch latest
- Deployment mode(standalone or cluster): both
- MQ type(rocksmq, pulsar or kafka):    all
- SDK version(e.g. pymilvus v2.0.0rc2): dev latest
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

 The "complete" state of the interface "get_compaction_state" does not include the index time for the new segments after major compaction

### Expected Behavior

 The "complete" state of the interface "get_compaction_state" should include the index time for the new segments after major compaction.

I think the new segments will replace the old segments when it is indexed completed, so the whole major compaction process should include the index process for the new segments.

 Or the benefits for the major compaction could not take effects.

**Comment by wayblink** [comment]
should fix in #32149 @binbinlv please verify

**Comment by binbinlv** [comment]
Fixed and verified.
milvus version: milvus-io-lru-dev-cff91b2-20240423
